pyvrp/tests/test_read.py

```diff
@@ -41,7 +41,8 @@ def test_reading_OkSmall_instance():
     # From the DIMENSION, VEHICLES, and CAPACITY fields in the file.
     assert_equal(data.num_clients, 4)
     assert_equal(data.num_vehicles, 3)
-    assert_equal(data.vehicle_capacity, 10)
+    for i in range(data.num_vehicles):
+        assert_equal(data.route(i).vehicle_capacity, 10)
```

Comment:
I don't like `i` as a variable name in general. Can we use `idx`, `vehicle_idx` or something like that instead?

Reply:
Feel free to change it, I don't think it's worth it.